docs/network_architecture.rst

```diff
@@ -259,31 +259,29 @@ Let's consider the basic lifecycle of state transitions in :math:`C`. Let
 
     graph TB
     A( ) -->|startup| B(STANDBY)
-    B -->|N Ps registered| C(ROUND i)
-    C -->|next round i := i+1| C
+    B -->|N registered| C(ROUND)
+    C -->|dropout| B
+    C -->|next round| C
     C -->|no more rounds| D(FINISHED)
 
-Once :math:`C` starts up, it's in the **STANDBY** state, waiting for incoming
```

Comment:
The previous version was slightly clearer imo.

Reply:
I think "waiting" over-used and there is even a Participant state called that now, so "open" is probably ok!